docs/validator/economics.md

```diff
@@ -0,0 +1,134 @@
+---
+id: economics
+title: The Economics of a Validator
+sidebar_label: Understanding Economics
+---
+## Overview
+
+NEAR is a sharded, developer-friendly, proof-of-stake blockchain. It will use a technique called 'dynamic sharding' that splits the network, so that much of the computation is done in parallel. With the increase of users and their transactions, NEAR will dynamically introduce additional shards.
```

Comment:
I think it's a bit strange to highlight something that we haven't implemented yet in the very beginning.